extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -0,0 +1,400 @@
+The opto-optical reference transform shall be applied, the reference transform compatible with both SDR and HDR displays is described in ITU BT.2390 with the use of range extension and gamma values.  
+
+E′ = G709[E] = pow(1.099 (rangeExtension * E), 0.45) – 0.099 for 1 > E > 0.0003024
+               267.84 * E for 0.0003024 ≥ E ≥ 0
+FD = G1886[E'] = pow(100 E′, gamma)
+
+E is the linear scene light value. 
+
+Where the rangeExtension and gamma values shall be set according to HDR or SDR display.  
+
+```
+HDR
+range extension = 59.5208  
+gamma = 2.4  
+```
+
+```
+SDR
+range extension = 46.42  
+gamma = 2.4  
+```
+
+
+**Implementation Notes**
+
+Pseudocode for BT.2100 reference OOTF  
+
+```
+BT_2100_OOTF(color, rangeExponent, gamma) {  
+    if (color <= 0.0003024) {  
+        nonlinear = 267.84 * color;  
+    else {  
+        nonlinear = 1.099 * pow(rangeExponent * color, 0.45) - 0.099;  
+    }  
+    return 100 * pow(nonlinear, gamma);
+}  
+```
+
+Where this is calculated per RGB component, note that the color value in this equation must be in range 0.0 - 1.0
+It may be estimated that the display will not have the ability to output dark levels in the region of 0.0003024 in which case implementations may ignore that condition and use the same operator for the whole range 0.0 - 1.0  
+
+The incoming color value shall first be adjusted by aperture as detailed in [See Scene aperture](#scene-aperture) 
+
+
+### OETF
+
+Opto-electrical-transfer-function  
+Input signal to the PQ OETF is the scene linear light. The PQ OETF maps scene linear light to non-linear PQ signal value.  
+For more information see the ITU BT.2100 specification.  
+
+After the OOTF is applied the OETF shall be applied, this will yield a non linear output-signal in the range [0.0 - 1.0] that is stored in the display buffer.  
+This shall be done according to the parameter `Reference PQ OETF`of ITU BT.2100    
+
+```
+Where the resulting non-linear signal (R,G,B) in the range [0:1] = ((C1 + C2 * pow(FD / 10000, m1)) / (1 + C3 * pow(FD / 10000, m1))  ^ m2 
+
+FD from the OOTF and  
+m1 = 2610/16384 = 0.1593017578125 
+m2 = 2523/4096 * 128 = 78.84375 
+c1 = 3424/4096 =0.8359375 = c3 − c2 + 1
+c2 = 2413/4096 * 32 = 18.8515625
+c3 = 2392/4096 * 32 = 18.6875
+```
+
+
+### Parameters
+
+The following parameters are added by the `KHR_displaymapping_pq` extension, it is added with the extension on a scene object:  
+
+| Name                   | Type       | Description                                    | Required           |
+|------------------------|------------|------------------------------------------------|--------------------|
+| **sceneAperture**   | `object`   | Scene light adjustment setting, if no value supplied the default value of OFF will be used and scene light values will be kept unmodified.    | No |
+
+
+### Scene aperture
```

Comment:
Hi Romain and thanks for your comments

Personally I feel that the meaning of the noun aperture is better suited for what this setting is doing.
Ie, the mechanism is only accounting for an opening that can be of varying size.

The word exposure is in my opinion coupled to shutter speed, ISO and aperture.

I would like to keep using aperture as a way to make a distinction from what could be a coming extension that defines a 'physical camera', with shutterspeed, ISO and aperture parameters.

Reply:
Aperture would conflict more with a potential extension defining aperture/shutter speed/ISO IMO. I've also never seen the word aperture used to describe what's effectively exposure.

Reply:
The parameter controls the virtual opening letting light into the observer (or the eye) with the goal of scaling down values when light contribution is higher than the dynamic range.
As such it has nothing to do with the exposure as used in a camera (aperture/shutter/ISO).

Based on this justification I feel that the most fitting term for this extension is aperture.
In my opinion exposure is more fitting for an extension that seeks out to model a physical camera having aperture/shutter and ISO.

Reply:
Exposure *is* the scaling of values. The fact that in a camera yo use aperture/shutter/ISO to control exposure doesn't mean exposure is those values (but you can compute a photometric exposure from those settings). It is so much more confusing to use the term "aperture" since it already has a specific meaning. I'm not advocating for the use of aperture/shutter/ISO here, just for using the term exposure.